config/monitoring/logging/elasticsearch/100-fluentd-configmap.yaml

```diff
@@ -26,16 +26,29 @@ data:
       root_dir /tmp/fluentd-buffers/
     </system>
   200.containers.input.conf: |-
+    # Capture logs from container's stdout/stderr -> Docker -> .log in JSON format
     <source>
-      @id fluentd-containers.log
+      @id containers-stdout-stderr
       @type tail
       path /var/log/containers/*user-container-*.log,/var/log/containers/*build-step-*.log,/var/log/containers/controller-*controller-*.log,/var/log/containers/webhook-*webhook-*.log,/var/log/containers/*autoscaler-*autoscaler-*.log,/var/log/containers/*queue-proxy-*.log,/var/log/containers/activator-*activator-*.log
-      pos_file /var/log/es-containers.log.pos
+      pos_file /var/log/containers-stdout-stderr.pos
       time_format %Y-%m-%dT%H:%M:%S.%NZ
       tag raw.kubernetes.*
       format json
       read_from_head true
     </source>
+    # Capture logs from Knative containers' /var/log
+    <source>
+      @id containers-var-log
+      @type tail
+      path /var/lib/kubelet/pods/*/volumes/kubernetes.io~empty-dir/knative-internal/**/*/**/*
```

Comment:
Could you add a comment explaining why `**/*/**/*` is used instead of `**/*`?

Reply:
Done.